roles/2_setup-infra/tasks/subtasks/dependencies-redhat.yml

```diff
@@ -18,6 +18,17 @@
   tags:
     - skip_ansible_lint
 
+- name: Install specified kernel
+  yum:
+    pkg: "kernel-{{ sp_kernel_version }}"
+    update_cache: yes
+  when:
+    - sp_kernel_version is defined
+    - not ansible_kernel | regex_search("^" + sp_kernel_version)
+
+- name: Include tasks for setting default kernel
+  include_tasks: "kernel.yml"
```

Comment:
Please use a more elaborate naming scheme, for example, `boot_into_updated_kernel` or something similar.